data_safe_haven/resources/workspace/workspace.cloud_init.mustache.yaml

```diff
@@ -75,65 +107,34 @@ apt:
 
 # Install necessary apt packages
 packages:
-  # System requirements
+  - ansible
   - libnss-ldapd  # LDAP login
   - libpam-ldapd  # LDAP login
   - ldap-utils    # LDAP login
   - nfs-common    # Mounting shared volumes
-  - xfce4         # XFCE desktop
-  - xrdp          # remote desktop client
-  # Programming
-  - python-is-python3  # Python language
-  - python3            # Python language
-  - python3-dev        # Python language
-  - python3-pip        # Python language
-  - python3-venv       # Python language
-  - r-base             # R language
-  # Tools
-  - firefox       # web browser
-  - libpq-dev     # interact with PostgreSQL databases
-  - msodbcsql17   # interact with Microsoft SQL databases
-  - unixodbc-dev  # interact with Microsoft SQL databases
-  # Bash testing
-  - bats
 package_update: true
 package_upgrade: true
 
 runcmd:
-  # Configure LDAP user login
-  # -------------------------
-  - echo ">=== Configure LDAP user login... ===<"
-  # Configure nsswitch
-  - echo "Adding ldap to /etc/nsswitch.conf"
-  - sed -i -r "s|^(passwd\|group\|shadow)(:.*)$|\1\2 ldap|g" /etc/nsswitch.conf
-  # Configure PAM for LDAP
-  - echo "Configuring PAM for LDAP login"
-  - sed -i "s|pam_mkhomedir.so|pam_mkhomedir.so skel=/etc/skel umask=0022|g" /usr/share/pam-configs/mkhomedir
-  - sed -i "s| use_authtok||g" /usr/share/pam-configs/ldap
-  - pam-auth-update --enable mkhomedir ldap
-  # Allow SSH password authentication
-  - sed -i "s|^PasswordAuthentication.*|PasswordAuthentication yes|g" /etc/ssh/sshd_config
-  - sed -i "s|^UsePAM.*|UsePAM yes|g" /etc/ssh/sshd_config
   # Restart services
-  - systemctl restart nscd
   - systemctl restart nslcd
-  - systemctl restart sshd
-
-  # Enable xrdp and session manager at startup
-  # ------------------------------------------
-  - echo ">=== Enable xrdp... ===<"
-  - systemctl enable xrdp
-  - systemctl enable xrdp-sesman
-  - systemctl start xrdp
-  - systemctl start xrdp-sesman
 
   # Mount all external volumes
   # --------------------------
   - echo ">=== Mounting all external volumes... ===<"
   - grep -v -e '^[[:space:]]*$' /etc/fstab | sed 's|^|  /etc/fstab  |'
   - mount -fav
   - while (! mountpoint -q /data); do sleep 5; mount /data; done
+  - while (! mountpoint -q /desired_state); do sleep 5; mount /data; done
   - while (! mountpoint -q /home); do sleep 5; mount /home; done
   - while (! mountpoint -q /output); do sleep 5; mount /output; done
   - while (! mountpoint -q /shared); do sleep 5; mount /shared; done
-  - findmnt
```

Comment:
What's the reason for dropping `findmnt`? Is it because we think sysadmins will need/want to look at these log files? Or is this done better somewhere else?

Reply:
An accident 😅.